css-layout-classes/index.html

```diff
@@ -0,0 +1,120 @@
+          mode was widely criticized.
+        </p>
+      </div>
+      <div class="column-one-third">
+        <input
+          type="image"
+          id="yoshi"
+          alt="NintendoCharacterYoshi"
+          src="images/yoshi.png" />
+      </div>
+      <div class="row">
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="kirby"
+            alt="NintendoCharacterKirby"
+            src="images/kirby.png" />
+        </div>
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="Fox"
+            alt="NintendoCharacterFox"
+            src="images/fox.png" />
+        </div>
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="pikachu"
+            alt="NintendoCharacterPikachu"
+            src="images/pikachu.png" />
+        </div>
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="ness"
+            alt="NintendoCharacterNess"
+            src="images/ness.png" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-half">
+          <h2>Donkey Kong</h2>
+          <p>
+            Donkey Kong is a series of video games featuring the adventures of a
+            gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+            The franchise consists mainly of two games genres, but also includes
+            additional spin-off titles of various genres.
+          </p>
+          <p>
+            The games of the first genre are mostly single-screen
+            platform/action puzzle types, featuring Donkey Kong as the opponent
+            in an industrial construction setting.
+          </p>
+        </div>
+        <div class="column-one-half">
+          <input
+            type="image"
+            id="dk"
+            alt="NintendoCharacterDK"
+            src="images/dk.png" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-half">
+          <input
+            type="image"
+            id="mario"
+            alt="NintendoCharacterMario"
+            src="images/mario.png" />
+        </div>
+        <div class="column-one-half">
+          <h2>Mario</h2>
+          <p>
+            Mario is a fictional character in the Mario video game franchise,
+            owned by Nintendo and created by Japanese video game designer
+            Shigeru Miyamoto. Acting ad the company's mascot, as well as being
+            the eponymous protagonist of the series, Mario has appeared in over
+            200 video games since his creation.
+          </p>
+          <p>
+            Depicted as a short, pudgy, Italian plumber who resides in the
+            Mushroom Kingdom, his adventures generally center upon rescuing
+            Princess Peach from the Koopa villain Bowser.
+          </p>
+        </div>
+      </div>
```

Comment:
You currently have all of your other `.row` elements stuffed inside of this one `.row`. All of your `.row` elements should be siblings. You shouldn't have any other `.row` elements inside of another `.row`